### PR TITLE
DOCSP-41952: Download and Install

### PR DESCRIPTION
**Pull request by norareidy**
JIRA - https://jira.mongodb.org/browse/DOCSP-41952
This PR also covers https://jira.mongodb.org/browse/DOCSP-41951 
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41952-download-install/get-started/download-and-install/
https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41952-download-install/get-started/

### CODE REVIEW

File: source/get-started/download-and-install.txt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      your ``php.ini`` file:
+
+      .. code-block:: bash
+
+         extension=mongodb.so
+
+      .. tip::
+
+         You can locate your ``php.ini`` file by running the following command
+         from your shell:
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:45]:
```suggestion
         the following shell command:
```

Comment (shuangela) [reply source/get-started/download-and-install.txt:45]: Or to create consistency: "running the following command in your shell"

File: source/get-started/download-and-install.txt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+         mkdir php-quickstart
+
+      Run the following commands to create a ``quickstart.php`` application file in
+      the ``php-quickstart`` directory:
+
+      .. code-block:: bash
+
+         cd php-quickstart
+         touch quickstart.php
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:66]: S: To my knowledge, the touch command only works on linux/mac machines. This thread: https://stackoverflow.com/questions/30011267/create-an-empty-file-on-the-commandline-in-windows-like-the-linux-touch-command provides alternatives that I think work on both Windows and Linux, unless we want to specify that this guide is only valid for mac/linux machines

File: source/get-started/download-and-install.txt [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+      To install the {+php-library+}, run the following command from your ``php-quickstart``
+      directory:
+
+      .. code-block:: bash
+
+         composer require mongodb/mongodb
+      
+      After installing the library, include Composer's ``autoload.php`` file by adding the
+      following line to ``quickstart.php``:
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:78]: Q: Do the users add this line at the end of the file, or at the beginning/does placement of the line matter?

File: source/get-started/download-and-install.txt [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+      .. code-block:: bash
+
+         require_once __DIR__ . '/vendor/autoload.php';
+
+After you complete these steps, you have a new project directory
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:84]:
```suggestion
After you complete these steps, you have a new project directory, a new application file,
```

File: source/get-started/download-and-install.txt [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure that you have the following
+      dependencies installed on your local machine:
+      
+      - `PHP <https://www.php.net/manual/en/install.php>`__
+      - `Composer <https://getcomposer.org/download/>`__
+    
+   .. step:: Install the MongoDB extension
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:25]:
```suggestion
   .. step:: Install the MongoDB PHP extension
```

File: source/get-started/download-and-install.txt [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+      Run the following command to install the ``mongodb`` PHP extension:
+      
+      .. code-block:: bash
+
+         sudo pecl install mongodb
+
+   .. step:: Update your PHP configuration file
+
+      To enable the ``mongodb`` extension, add the following line to the top of
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:35]:
```suggestion
      To enable the ``mongodb`` extension in your PHP configuration file, add the following line to the top of
```

File: source/get-started/download-and-install.txt [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure that you have the following
+      dependencies installed on your local machine:
+      
+      - `PHP <https://www.php.net/manual/en/install.php>`__
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:22]: Q: Which version of PHP/Composer are necessary?

**Review by shuangela — CHANGES_REQUESTED** [review]
Good start, some questions/things to think about for you!

File: source/get-started/download-and-install.txt (hunk 7 above)
Comment (norareidy) [reply source/get-started/download-and-install.txt:22]: added!

File: source/get-started/download-and-install.txt (hunk 3 above)
Comment (norareidy) [reply source/get-started/download-and-install.txt:78]: It should go at the top

File: source/get-started/download-and-install.txt [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+      Run the following command to install the ``mongodb`` PHP extension:
+      
+      .. code-block:: bash
+
+         sudo pecl install mongodb
+
+   .. step:: Update your PHP configuration file
+
+      To enable the ``mongodb`` extension in your PHP configuration file, add the
+      following line to the top of your ``php.ini`` file:
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:36]: Q: Does the php.ini file exist already once you download the dependency? Where would it be installed?

File: source/get-started/download-and-install.txt [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+             :tabid: create-file-windows
+
+             .. code-block:: bash
+
+                cd php-quickstart
+                type nul > quickstart.php
+
+   .. step:: Install the {+php-library+}
+
+      To install the {+php-library+}, run the following command from your ``php-quickstart``
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:83]:
```suggestion
      To install the {+php-library+}, run the following command in your ``php-quickstart``
```

**Review by shuangela — CHANGES_REQUESTED** [review]
one quick question but good work!

File: source/get-started/download-and-install.txt (hunk 8 above)
Comment (norareidy) [reply source/get-started/download-and-install.txt:36]: Yes, when you install PHP. You can see where it's installed with php --ini command (the admonition below) - for me, I used homebrew to install, so it's in /opt/homebrew/etc/php/8.3

**Review by shuangela — APPROVED** [review]
LGTM!

File: source/get-started/download-and-install.txt [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+   .. step:: Create a project directory
+
+      From your root directory, run the following command in your shell to create
+      a directory called ``php-quickstart`` for this project:
+
+      .. code-block:: bash
+
+         mkdir php-quickstart
+
+      Select the tab corresponding to your operating system and run following commands
```
Comment (shuangela) [inline source/get-started/download-and-install.txt:60]: Sorry, just noticed this! Quick fix:

```suggestion
      Select the tab corresponding to your operating system and run the following commands
```

**Review by shuangela — APPROVED** [review]

File: source/get-started/download-and-install.txt
Comment (GromNaN) [inline source/get-started/download-and-install.txt:1]: The [installation tutorial](https://www.mongodb.com/docs/php-library/current/tutorial/install-php-library/) is a duplicate after this page is created. Maybe there's no point in keeping 2 almost identical pages.

Comment (norareidy) [reply source/get-started/download-and-install.txt:1]: Agreed. Usually we have a cleanup ticket after finishing all the standardization tickets to remove the duplicate content, but happy to take out the Installation page in this PR

File: source/tutorial/install-php-library.txt [hunk 11]
```diff
@@ -1,99 +0,0 @@
...
-classes *and* functions are loaded for your application:
-
-#. If you are using a `PSR-4 <https://www.php-fig.org/psr/psr-4/>`_ autoloader,
-   map the top-level ``MongoDB\`` namespace to the ``src/`` directory. If you
-   are not using an autoloader, manually require _all_ class files found
-   recursively within the ``src/`` directory.
-
-#. Regardless of whether you are using an autoloader, manually require the
-   ``src/functions.php`` file. This is necessary because PHP does not support
-   autoloading for functions.
```
Comment (GromNaN) [inline source/tutorial/install-php-library.txt:99]: I agree that this process should no longer be documented. Composer is now widely used and not using it is reserved for experienced users who know how.

**Review by GromNaN — APPROVED** [review]

**Pull request merged**